types/react/package.json

```diff
@@ -180,6 +180,10 @@
         {
             "name": "Dmitry Semigradsky",
             "githubUsername": "Semigradsky"
+        },
+        {
+            "name": "Matt Pocock",
+            "githubUsername": "mattpocock"
```

Comment:
@mattpocock Are you also ok with me adding you as a contributor here? I seem to remember you being ok with but totally fine if you don't want the GH notification noise. You'd get pings on every PR targetting React. Doesn't mean you have to review. Just helps keeping you in the loop.

Reply:
Absolutely fine!